### ISSUE TITLE
Need version key in the manifest

### ISSUE THREAD

**Issue by mbriney** [nonsleepr/ha-eufy-security#40]
Seems like HA is going to require versioning starting in 2012.6

`No 'version' key in the manifest file for custom integration 'eufy_security'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'eufy_security'`

**Comment by ziptbm** [comment]
I'm seeing this as well on a fresh install.

**Comment by mawiak** [comment]
Already fixed in Master-Branch.
Select Masterbranch in HACS install (last entry in drop down List)

**Comment by htuomola** [comment]
Yeah I think it's because there is no release for v0.3.1 but there is one for 0.3.0, HACS is using the latest release. Even having no releases would be kinda better as it would default to master, I think. Of course adding release for 0.3.1 would be best.

**Comment by Dunc63** [comment]
> Already fixed in Master-Branch. Select Masterbranch in HACS install (last entry in drop down List)

Can you explain how I do this please? Do I need to uninstall the eufy_security component - not sure how to do that? I guess then I somehow select 'Masterbranch' during the reinstall in HACS?
